spoken/helpers.py

```diff
@@ -89,3 +89,59 @@ def get_home_events():
     events = Event.objects.filter(event_date__gte=today).order_by("event_date")[:2]
     cache.set(cache_key, events, timeout=CACHE_EVENTS)
     return events
+
+# ----  Tutorials List ----
+def get_tutorials_list(foss, lang):
+    foss_key = f"tutorials:{foss.lower().strip().replace(' ','_')}"
+    lang_key = lang.lower().strip()
+    cache_key = f"{foss_key}_{lang_key}"
```

Comment:
```suggestion
    cache_key = f"tutorials_list:{foss_key}_{lang_key}"
```
in the key, also put a string at the start to avoid clashes